test/integration/backward_compatible/parallel/serialization/DefaultSerializersLiveTest.js

```diff
@@ -664,3 +658,50 @@ describe('DefaultSerializersLiveTest', function () {
         }
     });
 });
+
+describe('DefaultSerializersLiveTest Arrays', function () {
+    let cluster, client;
+
+    const testFactory = new TestUtil.TestFactory();
+
+    before(async function () {
+        cluster = await testFactory.createClusterForParallelTests();
+        const member = await RC.startMember(cluster.id);
+        client = await testFactory.newHazelcastClientForParallelTests({
+            clusterName: cluster.id,
+            serialization: {
+                defaultNumberType: 'byte'
+            }
+        }, member);
+    });
+
+    beforeEach(async function() {
+        map = await client.getMap(TestUtil.randomString(10));
```

Comment:
Instead of reusing the same variable, I think we should define the variable twice in each describe block. Reusing the function is fine though

Reply:
Fixed.